docker-compose.yml

```diff
@@ -9,15 +9,15 @@ services:
     environment:
       - POSTGRES_USER=${DB_USERNAME}
       - POSTGRES_PASSWORD=${DB_PASSWORD}
-      - POSTGRES_DB=${DB_DATABASE}
-    ports:
-      - '5432:5432'
+      - POSTGRES_DB=${DB_NAME}
     # ports:
-    #   - ${DB_PORT}:${DB_PORT}
+    #   - '5432:5432'
+    ports:
+      - ${DB_PORT}:${DB_PORT}
     networks:
     - 'shared-network'
-    volumes:
-      - ./pgdata:/var/lib/postgresql/data
+    # volumes:
+    #   - ./pgdata:/var/lib/postgresql/data
   
   web:
```

Comment:
Good work. app or server would be a better service name than "Web"

Reply:
Yes I did it